libs/components/core/src/lib/modules/overlay/overlay.component.ts

```diff
@@ -71,6 +77,8 @@ export class SkyOverlayComponent implements OnInit, OnDestroy {
 
   public zIndex = `${++uniqueZIndex}`;
 
+  public clipPath$ = new ReplaySubject<string>(1);
```

Comment:
What's the implementation look like for this public property? Who uses it and how?

Reply:
The idea is that the consumer optionally [observes a container](https://github.com/blackbaud/skyux/pull/981/files#diff-4474a869bcdcf4cf0012698e9b09018cd69cbac99693bacaf633087e185303b7R236-R260) and then [pushes changes](https://github.com/blackbaud/skyux/pull/981/files#diff-4474a869bcdcf4cf0012698e9b09018cd69cbac99693bacaf633087e185303b7R102-R104) so that an overlay can behave like it is within a container element.